test/core.test.js

```diff
@@ -147,7 +163,6 @@ describe('core', () => {
     timer.tick(INTERVAL[1] * 1000)
     assert.deepEqual(agent._cache.a, {})
     assert.deepEqual(agent._cache.l, {})
-    assert(agent.active)
```

Comment:
Set active flag is move to onload callback.
So, can't evaluate at this point...